### ISSUE TITLE
[FEATURE] Allow collecting demographic and other information

### ISSUE THREAD

**Issue by richlv** [mjstarrett/Landmarks#16]
It would be great to have a prefab that would allow gathering demographic (age, sex etc) and other information at the beginning of the experiment. It would be saved along with the experiment results.

If this is possible already, very sorry for missing it - poked around a lot, didn't find anything like that in "Prefabs".
The closest was "TASK_SliderQuestions", but there did not seem to be a way to collect user responses.

**Comment by richlv** [comment]
The startup screen, visible in #20, does look somewhat as if able to provide similar functionality, but no clue how to possibly modify any of that, and whether that would be saved along with the experiment data.

**Comment by mjstarrett** [comment]
Yes, that scene is where demographics would be added. Given that I haven't put together documentation for how to use the startup scene yet, I have removed it from the default build settings. This is a planned feature, and hopefully I will add it and document the startup scene process soon.

**Comment by richlv** [comment]
Thank you so much, that sounds great. Happy to hep with testing or documentation.
